tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_prplwrt.py

```diff
@@ -67,3 +104,36 @@ def touch(self):
         Purpose is to keep consoles active, so they don't disconnect for long running activities.
         """
         pass
+
+    def get_docker_subnet(self, docker_network: str) -> IPv4Network:
+        """Get subnet used by docker network."""
+        docker_network_inspect_cmd = ('docker', 'network', 'inspect', docker_network)
+        inspect_raw = subprocess.run(docker_network_inspect_cmd, stdout=subprocess.PIPE)
+        if inspect_raw.returncode != 0:
+            # Assume network doesn't exist yet. Create it.
+            # Raise an exception if it fails (check=True).
+            subprocess.run(('docker', 'network', 'create', docker_network), check=True,
+                           stdout=subprocess.DEVNULL)
+            # Inspect again, now raise if it fails (check=True).
+            inspect_raw = subprocess.run(docker_network_inspect_cmd, check=True,
+                                         stdout=subprocess.PIPE)
+        inspect_json = json.loads(inspect_raw.stdout)
+
+        return IPv4Network(inspect_json[0]["IPAM"]["Config"][0]["Subnet"])
+
+    def set_iface_to_bridge(self, iface: str) -> bool:
+        """Add specified interface to the specified bridge."""
+        ip_command = ("ip link set {} master br-lan".format(iface))
+        self.sendline(ip_command)
+        self.expect(self.prompt, timeout=10)
+
+    def set_iface_ip(self, iface: str, ip: IPv4Address, prefixlen :int) -> bool:
+        """Set interface IPv4 address."""
+        self.sendline("ip a add {}/{} dev {}".format(ip, prefixlen, iface))
+        self.expect(self.prompt, timeout=10)
```

Comment:
> Although even so, I don't see why it would be needed to add the IP address. No IP connectivity is needed between the devices, so why bother?
Do we require IP connectivity between agent and controller, don't we?

Reply:
No we don't. MultiAP is purely on Ethernet level.